src/components/SearchButton/SearchButton.tsx

```diff
@@ -0,0 +1,38 @@
+import clsx from 'clsx';
+import React, { MouseEventHandler } from 'react';
+import styles from './SearchButton.module.css';
+import Button from '../Button';
+
+export type SearchButtonProps = {
```

Comment:
Is there value in just using `Button` props? Or would that be not advised since we don't other props such as `variant` and `status` to be used here?

Reply:
With so few props with such generic types, I feel like this is easier.